chia/server/server.py

```diff
@@ -377,6 +382,10 @@ async def incoming_connection(self, request):
     async def connection_added(self, connection: WSChiaConnection, on_connect=None):
         # If we already had a connection to this peer_id, close the old one. This is secure because peer_ids are based
         # on TLS public keys
+        if connection.closed:
+            self.log.debug(f"Curious: ignoring request to add closed connection {connection.peer_host} ")
```

Comment:
Maybe?  Or am I being too business serious here...

```suggestion
            self.log.debug(f"ignoring unexpected request to add closed connection {connection.peer_host} ")
```